### ISSUE TITLE
Is it possible to deal with the equality constraints with the proposed miosqp?

### ISSUE THREAD

**Issue by Yangli0505** [oxfordcontrol/miosqp#7]
I have to deal with the equality constraints, but there are no options in the miosqp model and the inputs of setup function only include P, q, A, l, u, i_idx, i_l, i_u. Then, how to deal with equality constraints？If I set l=u, the l≤Ax≤u, is it okay?

**Comment by goulart-paul** [comment]
Yes, just set the lower and upper bounds equal.   The solver will recognise that as an equality and act accordingly.

**Comment by Yangli0505** [comment]
thanks!
祝好！

李洋
清华大学汽车工程系
Department of Automotive Engineering,
Tsinghua University
Email Address：lyxc56@gmail.com
Google scholar:https://scholar.google.com/citations?user=jsO5m0UAAAAJ&hl=en


Paul Goulart <notifications@github.com> 于2020年6月16日周二 下午6:10写道：
